WPFDirector/MainWindow.xaml.cs

```diff
@@ -11,16 +14,27 @@ namespace WPFDirector
     /// </summary>
     public partial class MainWindow : Window
     {
+        IUserServerController userServerController = new UserServerController();
         public MainWindow()
         {
             InitializeComponent();
         }
 
-        private void hospitalMapPressed(object sender, RoutedEventArgs e)
+        private void LoginClick(object sender, RoutedEventArgs e)
         {
-            HospitalMainWindow window = HospitalMainWindow.GetInstance(Role.Director);
-            this.Close();
-            window.ShowDialog();
+            LoggedUser user = new LoggedUser(emailTextBox.Text, passwordTextBox.Password);
+            LoggedUser.Role = Role.Director;
```

Comment:
You can allow a role to be defined in the constructor only. That would be better because it will not be possible to change the role in the rest of the program.